src/services/token.service.js

```diff
@@ -0,0 +1,40 @@
+import { Token } from '../models/token.js';
+
+async function save(userId, newToken) {
+  const token = await Token.findOne({ where: { userId } });
+
+  if (!token) {
+    await Token.create({ userId, refreshToken: newToken });
+
+    return;
+  }
+
+  token.refreshToken = newToken;
+  await token.save();
+}
+
+function getByToken(refreshToken) {
+  return Token.findOne({ where: { refreshToken } });
+}
+
+async function remove(userId) {
+  try {
+    const token = await Token.findOne({ where: { userId } });
+
+    if (!token) {
+      return { success: false, message: 'Token not found' };
```

Comment:
Consider logging the 'Token not found' message for debugging purposes. This can help track down issues related to token management.